### DISCUSSION TITLE
Thoughts on a webhook for actions?

### DISCUSSION THREAD

**Discussion by mcdowellster** [OliveTin/OliveTin#130] [Ideas]
I'd love to slap a link in documentation that can be clicked which can run an action instead of doing a post method :)

**Comment by jamesread** [comment]
Hey, I think #70 covers this - could you add your comments there? I'm interested to know how people would use this / what they need.

**Comment by mcdowellster** [comment]
It's mostly about simply running an action without the interface OR curl.
Ideally I'd have a link say: http://localhost:1337/action?sending

Simple and totally amazing on a document where an end-user could run a
command without "running a command" :)

**Comment by jamesread** [comment]
This is now implemented :-) 

https://docs.olivetin.app/api-start-action.html#api-eg-startIdUrl
